# Request
bool image_request

# Response
---
sensor_msgs/Image image_data
string message